### DISCUSSION TITLE
RAD issues on Drean C64

### DISCUSSION THREAD

**Discussion by BizarreKING** [frntc/RAD#27] [General]
I have had several problems with the RAD in my C64 which is the Argentine version also known as Drean, this C64 model is a PAL-N standard and for that it uses the VIC-II 6572 which was the specific version for that region.

To begin with, the screen with the main menu looks full of graphic glitches and that happens with both the official RAD software and the Doom demo, I rule out that it is a c64 failure or a Raspberry power problem since It is externally powered without problems with an original charger.

After that, the other problem that it has is that the PRG roms are placed directly on the micro sd, despite the fact that they appear in the menu, none of them load, none of them work, and finally, I have tried many times to load sonic and it always gives errors. It loads at different points (it's not a problem with the micro sd either, I already tried with 2 and the result is the same).

I have tested all of the above on a C128D that has NTSC standard and I had no problems (except the doom demo because it is only compatible with PAL) so I conclude that the RAD has serious compatibility problems with the VIC-II 6572.

I would appreciate if this could be fixed in future firmware updates as well as being able to add a whole library of games in PRG format and if it were possible for it to be compatible with other formats such as CRT.
In any case, I congratulate you for your work with the RAD and I am sure that you will be able to improve it even more with time.

Greetings from South America.

**Comment by frntc** [comment]
Greetings to South America!

In fact I have no chance to test on a Drean C64, as they are close to impossible to get here in Europe (if you know where to get Drean C64s or VIC-II 6572s let me know, I'm looking to them for testing my projects!).

Apart from that: I'm pretty sure that it is only a matter of finding the correct timing values. Have you tried the timing menu (pound key) and changing them? Most likely only the last 3 values need to be fixed. As long as the main menu has glitches, it's also very likely that any DMA transfer (including PRG launching) does not work reliably.

**Reply by BizarreKING** [reply]
Maybe I could provide you with a VIC-II PAL-N with its corresponding crystal, if you're interested, write me to bizarre_king@live.com and we'll talk about it.

Thanks in advance.

**Comment by odflor** [comment]
Only for the records,...I'm also from South América (Paraguay).

I ain't find the correct timing values to use my RAD with my c128 NTSC (slim).

To the date, I can't see any of the nuvies or any other thing.

But still trying,... some day, I can catch the correct values.

**Reply by frntc** [reply]
Nuvies will not work on NTSC machines! Playing Nuvies requires precise timings, which in turn depend on whether the VIC is PAL or NTSC (and they are made for PAL machines). Do other REU things work on your C128 NTSC?

edit: btw. cool to see that RADs made it thus far :-)

**Reply by odflor** [reply]
Mr. frntc:

Nope, no other REU features work for me on my C128 NTSC. I think is because I ain't catch the correct timing values.

I'm a user of the 1541u2+, but I choose to investigate the RAD/SideKick64 project for the possibility to extend them for execute custom programs on the RaspberryPI that serve commodore programs as extends add-ons.

Now I'm learning GCC to know how to read and understand the source of both projects.

Thanks for Your precisions and a lot of thanks for open Your project to the world.

Best Regards

**Reply by frntc** [reply]
Two questions to diagnose the problem: 1) does the menu work and in case there are glitches, can you adjust the timings such that they are gone? 2) if the menu works, can you run a GeoRAM-test? (e.g. this one: http://www.cbmhardware.de/dlmanager/index.php?id=444)

**Reply by odflor** [reply]
Hi!

To save stress I use the RAD on a C64 breadbin NTSC and all works (except Nuvie).

I test my RAD on a C64C PAL of my friend and ALL works.

Thank You very much for your support.

Best Regards.
